core/station/impl/src/services/account.rs

```diff
@@ -553,7 +553,10 @@ impl AccountService {
                                 CallerGuard::new(
                                     state.clone(),
                                     balance_update_guard_key,
-                                    Some(time() + Duration::from_secs(5 * 60).as_nanos() as u64),
+                                    CallerGuardParams {
+                                      max_concurrency: None,
+                                      expires_at_ns: Some(time() + Duration::from_secs(5 * 60).as_nanos() as u64)
+                                    },
```

Comment:
nit: it would be nice to have this with a builder pattern and derive default, this way you would be able to write this as

```suggestion
CallerGuardParams::default()
  .with_expires_at_ns(time() + Duration::from_secs(5 * 60).as_nanos() as u64),
```